analyzer/src/test/kotlin/managers/utils/DotNetSupportTest.kt

```diff
@@ -32,9 +32,7 @@ import java.io.File
 import java.time.Instant
 
 class DotNetSupportTest : StringSpec() {
-    private val projectDir = File("src/funTest/assets/projects/synthetic/dotnet").absoluteFile
-
-    init {
+      init {
```

Comment:
This should also remove `import java.io.File`, and the test could now use the `StringSpec({ ... })` syntax to get rid of the `init` block and reduce nesting.

Reply:
Both done.